.gitpod.yml

```diff
@@ -0,0 +1,7 @@
+tasks:
+  - init: |
+      npm install
+      gp sync-done installation
+    command: npm run lint
+  - init: gp sync-await installation
+    command: npm run build && npm run start
```

Comment:
Hi 👋 Just one more thing is missing before we can merge this PR, you need to add prebuilds, and that's it.

Reply:
Sure 👍 I'm first time writing Gitpod file 😅